tests/testthat/setup-run-model.R

```diff
@@ -101,8 +101,7 @@ naomi_evaluate_promise <- function (code, print = FALSE) {
   if (result$visible && print) {
     withr::with_output_sink(temp, print(result$value))
   }
-  output <- paste0(readLines(temp, encoding = "UTF-8", warn = FALSE),
-                   collapse = "\n")
+  output <- paste0(brio::readLines(temp), collapse = "\n")
```

Comment:
This line is failing in tests, https://buildkite.com/mrc-ide/naomi/builds/433#71cdd1ff-81c4-48c9-b778-3b1a3b20ef9c/19-927 

`temp` here is a file connection, `brio::read_file` might work here.